src/app/view/perspectives/perspective.ts

```diff
@@ -21,7 +21,6 @@ export enum Perspective {
   Detail = 'detail',
   Search = 'search',
   Table = 'table',
-  PostIt = 'postit',
```

Comment:
I think we can keep it here if it still needs to be referenced somewhere. We just need to add it to `FilterPerspectivesPipe` besides `SmartDoc`.

Reply:
It is only needed in this file, so I think it's better to remove it from `Perspective` enum.